tests/RateLimitErrorTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Stripe;
+
+class RateLimitErrorTest extends TestCase
+{
+    private function rateLimitErrorResponse()
+    {
+        return array(
+            'error' => array(),
+        );
+    }
+
+    public function testRateLimit()
+    {
+        try {
```

Comment:
This will not fail if the exception is not thrown. PHPUnit supports an `expectedException` annotation which will help: https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.exceptions.examples.ExceptionTest.php